### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Ashishgupta08**
### E-Commerce App 
You can check the live demo of the app [here](https://100steps.netlify.app/)

Please give your valuable feedback for the enhancement of my project.
ThankYou!!!

### CODE REVIEW

**Comment by Akshay-Laxmipathi** [comment]
Try adding the items to wishlist without having to redirect to the wishlist page everytime you click on the add to wishlist button.

File: src/Pages/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        <Nav />
+        <div className="home-page">
+            <div className="header">
+                <div className="header-bg"></div>
+                <div className="header-text">
+                    <p className="txt-1">DO IT NOW, RUN ON AIR</p>
+                    <p className="txt-2">RUNNING SHOES</p>
+                    <NavLink to='/products'>
+                        <button><strong>SHOP NOW</strong></button>
+                    </NavLink>
```
Comment (JatinRanka) [inline src/Pages/Home.jsx]: Use cursor pointer

File: src/components/Nav/Nav.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+export function Nav() {
+    return (
+        <>
+        <nav className="nav">
+            <div className="nav-header">
+                <h1>MYNTRA</h1>
+            </div>
+            <div className="nav-links">
+                <ul>
+                    <li><NavLink to='/' className="link">
```
Comment (JatinRanka) [inline src/components/Nav/Nav.jsx]: Add active class in nabber

File: src/Pages/Products.jsx [hunk 3]
```diff
@@ -1,36 +1,83 @@
...
 import './assets/css/product.css'
-import {Nav} from '../components/Nav/Nav'
+import { Nav } from '../components/Nav/Nav'
 import { FaSlidersH } from "react-icons/fa";
 import { BiSearch } from "react-icons/bi";
 import { IoHeartOutline } from "react-icons/io5";
 import { VscClose } from "react-icons/vsc";
+import LoadingScreen from "react-loading-screen";
+import { useCart } from "../Contexts/cart-context";
+import { useWishlist } from "../Contexts/wishlist-context";
```
Comment (JatinRanka) [inline src/Pages/Products.jsx:8]: Use index file for import

File: src/Pages/Products.jsx [hunk 4]
```diff
@@ -1,36 +1,83 @@
...
+  const [loader, setLoader] = useState(false)
+
+  function reducer(state,action){
+      switch (action.type) {
+        case "LOAD":
+          return {productsList: action.payload}
+        case "LOWTOHIGH":
+          return {productsList: state.productsList.sort((a,b)=>a.price.selling - b.price.selling)}
+        case "HIGHTOLOW":
+          return {productsList: state.productsList.sort((a,b)=>b.price.selling - a.price.selling)}
```
Comment (JatinRanka) [inline src/Pages/Products.jsx]: use snake case

**Review by JatinRanka — CHANGES_REQUESTED** [review]

File: src/Pages/Home.jsx (hunk 1 above)
Comment (Ashishgupta08) [reply src/Pages/Home.jsx]: Done

File: src/components/Nav/Nav.jsx (hunk 2 above)
Comment (Ashishgupta08) [reply src/components/Nav/Nav.jsx]: Done

File: src/Pages/Products.jsx (hunk 3 above)
Comment (Ashishgupta08) [reply src/Pages/Products.jsx:8]: Done

File: src/Pages/Products.jsx (hunk 4 above)
Comment (Ashishgupta08) [reply src/Pages/Products.jsx]: Done

**Comment by ishanjirety** [comment]
The overall app looks amazing design is elegant there are some minimal points that must be considered 

- Must update the title of the application
- instead of console logging error, try adding user interaction eg. Toast, inline errors.
- avoid using short-forms in CSS classes eg. txt-1,txt-2, it is preferred to use heading sub-heading as it improves readability 
- Add 404 pages to handle unmatched routes.

Else everything looks good.

**Pull request closed without merging**